CollAction/Frontend/app/project/ProjectList.tsx

```diff
@@ -106,6 +106,9 @@ class ProjectThumb extends React.Component<IProject, IThumbState> {
     }
   }
 
+  openProject = () => {
+    window.location.href = `/projects/${this.props.projectNameUriPart}/${this.props.projectId}/details`;
+  }
```

Comment:
Newline between functions

Reply:
fixed in 298caee

Reply:
No, what I meant that there has to be a newline between functions. Your fix removed the newline at a different position.